internal/faustjs.org/docs/reference/WordPressBlocksProvider.mdx

```diff
@@ -0,0 +1,71 @@
+---
+slug: /reference/WordPressBlocksProvider
+title: WordPressBlocksProvider Reference
+description: Reference docs for the WordPressBlocksProvider Component in Faust.
+---
+
+`WordPressBlocksProvider` is a [provider](https://www.patterns.dev/posts/provider-pattern/) component that wraps your entire app. This provider gives the proper context (which blocks have been defined as apart of your Faust app) to the `WordPressBlocksViewer` component.
+
+## Usage
+
+The below example shows how you can setup the `WordPressBlocksProvider` in your Faust app's `_app.js` file:
+
+```jsx {4,13-16,18} title="pages/_app.js"
+import '../faust.config';
+import React from 'react';
+import { useRouter } from 'next/router';
+import { WordPressBlocksProvider } from '@faustwp/blocks';
+import { FaustProvider } from '@faustwp/core';
+import blocks from '../wp-blocks/index.js';
```

Comment:
Mention what are blocks exactly here as the user does not have a reference.

Reply:
@theodesp Added this in the ["props" section](https://h2rry3jnw8kai20uqyyy0kmew.js.wpenginepowered.com/docs/reference/WordPressBlocksProvider#props), let me know if this works.